types/redact-axios-error/axios-error-groomer.d.ts

```diff
@@ -0,0 +1,15 @@
+export class AxiosErrorGroomer {
+  /**
+   * @param includeRequestData default true, false redacts request data
+   * @param includeResponseData default true, false redacts response data
+   * @param includeQueryData default true, false redacts query string data
+   */
+  constructor(includeRequestData?: boolean, includeResponseData?: boolean, includeQueryData?: boolean);
+
+  includeRequestData(flag: boolean): void;
+  includeResponseData(flag: boolean): void;
+  includeQueryData(flag: boolean): void;
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  getGroomedAxiosError(err: any): any;
```

Comment:
```suggestion
  getGroomedAxiosError(err: Error | null | undefined): Error | null | undefined;
```
https://github.com/tcollinsworth/redact-axios-error/blob/master/redact-axios-error.js#L29
you can probably do that way. Please verify with your Axios code if this looks OK.